luigi/parameter.py

```diff
@@ -222,6 +257,23 @@ def parse(self, x):
         """
         return x  # default impl
 
+    def parse_list(self, xs):
+        """
+        Parse a list of values from the scheduler.
+
+        Only possible if this parameter has a batch method. This will combine the list into a single
+        parameter value using batch method.
+
+        :param xs: list of values to parse and combine
+        :return: the combined parsed values
+        """
+        if self._batch_method is None:
+            raise NotImplementedError('No batch method found')
+        elif not xs:
+            raise ValueError('Empty parameter list passed to parse_list')
+        else:
+            return self._batch_method(map(self.parse, xs))
```

Comment:
About if `batch_method=list` will work or not:

@daveFNbuck, you've really got me confused. Here it really looks like that `xs` is one iterable of values. Ignoring the `parse`-step. If we have `xs=iterable(1,2,3)`, then as I see it:

 * `batch_method=max` ==> `3`
 * `batch_method=min` ==> `1`
 * `batch_method=','.join` ==> `'1,2,3'` (string)
 * `batch_method=list` ==> `[1,2,3]` (list of int)
 * `batch_method=(lambda x: x)` ==> `iterable(1,2,3)`

If it was this simple it would be fantastic. Because we wouldn't need `batch_tuple_parameter`. But that's not the case huh?